src/views/MainView.js

```diff
@@ -55,6 +55,9 @@ class MainView extends PureComponent {
 
   componentWillUnmount() {
     clearInterval(timerID);
+
+    const html = document.documentElement;
```

Comment:
Should we use getter here?

Reply:
Haha I asked Hugh this too, and he said that we know that the document will always be there so there is no need to use a getter.

Reply:
Ya, this code is safe. But usually YES you should use a GETTER